DIA land

VerJane23 opened the pull request:
The landing page project for the strategic agency AIR was completed using the BEM website development methodology and using SASS preprocessors. Built-in effects use smooth transitions, as well as elements of transformation and animation. Grid placement is mainly used to position project elements.
The project page is located at:  https://verjane23.github.io/layout_dia/

etojeDenys reviewed (changes requested):
1. remove the border
<img width="491" alt="image" src="https://github.com/mate-academy/layout_dia/assets/70220667/2fa22e23-c57a-4696-85ab-31f8aa7ced5c">


2. you don't need to add the cursor here, cursor represents how the icon must look on hover
<img width="171" alt="image" src="https://github.com/mate-academy/layout_dia/assets/70220667/cf8e47c7-0826-4e8b-b21e-c13401b0255e">

3. looks strange on big screens try to fix it
<img width="963" alt="image" src="https://github.com/mate-academy/layout_dia/assets/70220667/563922f2-3c9b-4f7f-b7d9-4b6b8b7647e2">

VerJane23 commented:
Denis, thank you for your support here and in the chat! Notes taken into account: - - the boundaries of the form fields have been removed; - - stretch the background to fill the entire screen; - - fixed cursor icon in the contacts field for hover;

anastasiiavorobiova reviewed (changes requested):
Great work! There are a few issues to fix: 
<img width="883" alt="Screenshot 2023-12-19 at 13 46 48" src="https://github.com/mate-academy/layout_dia/assets/135232001/a12962f8-70ff-40d1-9121-93442ccfbb86">
<img width="1057" alt="Screenshot 2023-12-19 at 13 46 55" src="https://github.com/mate-academy/layout_dia/assets/135232001/874002e2-e786-4ec3-838f-1ac6abbaa54d">
Consider fixing validation errors 
<img width="1256" alt="Screenshot 2023-12-19 at 13 45 59" src="https://github.com/mate-academy/layout_dia/assets/135232001/62f6c9ac-05ba-4579-bc63-5aa3f17a8d88">
The layout seems broken for wide screens 
<img width="580" alt="Screenshot 2023-12-19 at 13 43 23" src="https://github.com/mate-academy/layout_dia/assets/135232001/4c6475eb-159d-4a0e-b8e7-35a8e758a2d5">
Icons should be placed horizontally here 
<img width="589" alt="Screenshot 2023-12-19 at 13 42 30" src="https://github.com/mate-academy/layout_dia/assets/135232001/6980d7f1-d6cf-4a15-bca7-3c272581bbe7">
Consider opening a menu on click (you can use styles from the footer). The button should span the whole width of the screen 
<img width="450" alt="Screenshot 2023-12-19 at 13 41 42" src="https://github.com/mate-academy/layout_dia/assets/135232001/7beedbc0-9ec6-44b2-98a1-cdb19b9c5688">
Consider removing the default border from the button, and outline from the text area/inputs. Also, a user shouldn't be able to resize the text area. 
<img width="1082" alt="Screenshot 2023-12-19 at 13 41 15" src="https://github.com/mate-academy/layout_dia/assets/135232001/a60ca38c-9aff-454b-a372-3ba0beda3ea3">
Check margins here

VerJane23 commented:
Thank you, Anastasia, you make my code more perfect! Changes have been made: -- the background in section No. 4 has been expanded; -- changes have been made to the type attribute for the link and text area elements; -- the social media container was returned to a horizontal position; -- the text area field was brought into line with the requirements of the task - 2 lines were set, in addition, each line was set to 25 characters; -- the main block button of the mobile version now occupies the entire width of the screen; -- Clicking on the menu icon in versions for small screens moves the user and offers to use the footer menu links; -- Regarding the layout for wide screens, in accordance with Denys’s recommendations, I have already corrected this in the page block and now corrected it again in the slider block; on my widescreen monitor this layout is now stretched as it should. If you still have a broken view, please clarify how to fix it. Thank you in advance.

Viktor-Kost reviewed (changes requested):
Nice!

To improve:

1.1. write a placeholder for email similar to other ones ("Your email")
1.2 remove default input/button border/outline
1.3 add a little bit more rows for text area so it'll be easier to reread
<img width="616" alt="Screenshot 2023-12-19 at 18 32 51" src="https://github.com/mate-academy/layout_dia/assets/117587463/0969596c-e7d8-43ea-931a-b5ba8492f468">

2. make hover effects be the same for all social network links
https://github.com/mate-academy/layout_dia/assets/117587463/fda5bca3-75b8-4d09-ba4e-5c235e38666d

3. make transition be synchro. Now the link color changes faster than line
https://github.com/mate-academy/layout_dia/assets/117587463/06d8ab10-f9b9-4c07-9477-ddbb244eb268

4. button should open a mobile menu
<img width="354" alt="Screenshot 2023-12-19 at 18 39 51" src="https://github.com/mate-academy/layout_dia/assets/117587463/d7c21fef-adeb-42e2-9765-70d3acb0aa0f">

5. you shouldn't hardcode and limit the width for block for mobile screens. The should fulfill available space

https://github.com/mate-academy/layout_dia/assets/117587463/183f2f95-9a3e-4dd9-865a-9f6404741bf7

VerJane23 commented:
Thank you for your feedback, Victor! The following adjustments have been made: --two more lines have been added to the text area; --Your email is indicated in the field; -- removed outlines; -- synchronized effects;
-- an additional effect has been added to the Twitter logo; -- regarding the menu, initially this option was intended and developed by me, but it was absent from the design and did not find application in my work. Thank you for your persistent advice, now I have implemented the menu according to my own design.

etojeDenys reviewed (approved):
good job
